### PR TITLE
ch/proc version build url 1gmdpak

### PR DESCRIPTION
**Pull request by hoffmanc**

### CODE REVIEW

**Comment by zacnelson** [comment]
image

Currently seeing this when clicking on a completed Pico Work Order. I think the build URL needs more line wrapping as it's currently pushing the first column (maybe?) too far to the right.

**Comment by zacnelson** [comment]
> Currently seeing this when clicking on a completed Pico Work Order. I think the build URL needs more line wrapping as it's currently pushing the first column (maybe?) too far to the right.

Not sure how to control what shows up in this pop-up. Other than that, this is great.

**Review by zacnelson — APPROVED** [review]
Functionally working for me.

File: pico_mrp/data/activity_data.xml [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
 
+    <record id="mail_activity_type_missing_produce_serial" model="mail.activity.type">
```
Comment (sfenig) [inline pico_mrp/data/activity_data.xml]: is the activity stuff working?  thought you couldn't get that sorted?

**Review by sfenig — APPROVED** [review]
Nice man.  Thanks for cleaning this up and adding some more scripts/docs.  I thought the activity stuff ended up being a no go.    Did you get that working?  Can we get it going generically for the unknown error case as well, then?

File: pico_mrp/data/activity_data.xml (hunk 1 above)
Comment (hoffmanc) [reply pico_mrp/data/activity_data.xml]: no I couldn't get it working - I will remove this

**Pull request merged**